mcarch/views/mods.py

```diff
@@ -14,20 +14,24 @@
 
 @modbp.route("/mods")
 def browse():
-    mod_query = Mod.query.filter_by(draft=False)
+    drafts_only = request.args.get('drafts', type=bool) and cur_user().has_role(roles.archivist)
```

Comment:
When logged out, this raises `AttributeError: 'NoneType' object has no attribute 'has_role'`.